### ISSUE TITLE
How to make it support Chinese

### ISSUE THREAD

**Issue by cwcw** [doakey3/GreaseWriter#1]
This plugin is great
I hope it can support Chinese and custom start and end time

**Comment by doakey3** [comment]
I think Chinese would be a too hard since there are so many Chinese characters (one for each word right?). So that would mean creating a curve for thousands of characters. It could be possible if you found a single-line font (like [Hershey's single line fonts](https://www.evilmadscientist.com/2011/hershey-text-an-inkscape-extension-for-engraving-fonts/)) for Chinese that you could convert to curves in Blender.

**Comment by miklobit** [comment]
Can you add a short recipe to "readme" how to add a new font ?

**Comment by doakey3** [comment]
It's been a while since I looked at the code, but adding a new font looks like it would involve making changes to the `__init__.py` file and to the operators/write.py file. You would also have to create a font in the operators/fonts folder. You could look at the other fonts as an example. each font is a collection of symbols and each symbol is a text file (.glyph) with an ordered arrays of [x, y, z] coordinates that describe how to draw the symbol. Each line in the glyph represents a part of the symbol (for example a "?" has an upper squiggly and a lower dot, so there are 2 lines in the glyph file.)

To make the consolas font, I typed out each symbol in blender, then traced them with bezier curves, converted the curves to mesh, then used a python script to output the location of each vertex. It was a cumbersome task and I don't think I'd do it again.

In summary, there is no short recipe for adding a new font. It's a pain in the butt.
